### PR TITLE
zh-cn: Sync input/week with newest EN document

### PR DESCRIPTION
**Pull request by jasonren0403**
Finish translation of remaining English paragraph and remove `page` macro on page

### CODE REVIEW

File: files/zh-cn/web/html/element/input/week/index.md [hunk 1]
```diff
@@ -4,147 +4,151 @@ slug: Web/HTML/Element/Input/week
...
-| [`max`](#max)           | 接受为有效输入的最新年份和星期                       |
-| [`min`](#min)           | 最早的年和周接受为有效输入                           |
-| [`readonly`](#readonly) | 一个布尔值（如果存在），指示用户无法编辑字段的内容   |
-| [`step`](#step)         | 用于用户界面和约束验证的步进间隔（允许值之间的距离） |
+除了 {{HTMLElement("input")}} 元素共有的属性外，星期输入还提供以下属性：
 
 ### max
 
-接受以上 [值](#值) 部分中讨论的字符串格式的最新（按时间）年份和星期数。 如果输入到元素中的 {{htmlattrxref("value", "input")}} 超过此值，则元素将无法通过[约束验证](/zh-CN/docs/Web/Guide/HTML/HTML5/Constraint_validation)。 如果`max`属性的值不是有效的周字符串，则该元素没有最大值。
+接受以上 [值](#值) 部分中讨论的字符串格式的按时间最新年份和星期数。如果输入到该元素中的 {{htmlattrxref("value", "input")}} 超过此值，则元素将无法通过[约束验证](/zh-CN/docs/Web/Guide/HTML/Constraint_validation)。 如果 `max` 属性的值不是有效的星期字符串，则该元素没有最大值。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/week/index.md:78]:
```suggestion
接受以上[值](#值)部分中讨论的字符串格式的按时间最新年份和星期数。如果输入到该元素中的 {{htmlattrxref("value", "input")}} 超过此值，则元素将无法通过[约束验证](/zh-CN/docs/Web/Guide/HTML/Constraint_validation)。 如果 `max` 属性的值不是有效的星期字符串，则该元素没有最大值。
```

File: files/zh-cn/web/html/element/input/week/index.md [hunk 2]
```diff
@@ -4,147 +4,151 @@ slug: Web/HTML/Element/Input/week
...
+
+_目前，尚不清楚当与 `week` 输入一起使用时，`"any"` 的值对 `step` 意味着什么。确定该信息后，将对其进行更新。_
 
 ## 使用星期输入
 
-乍看之下，周输入听起来很方便，因为它们提供了用于选择周的简单 UI，并且它们标准化了发送到服务器的数据格式，而与用户的浏览器或区域设置无关。 但是， `<input type="week">` 存在问题，因为不能保证所有浏览器都支持浏览器。
+乍看之下，星期输入听起来很方便，因为它们提供了用于选择星期的简单 UI，并且标准化了发送到服务器的数据格式，而与用户的浏览器或区域设置无关。但是，`<input type="week">` 存在问题，因为不能保证所有浏览器都支持该特性。
 
-我们将研究 `<input type="week">`的基本和更复杂的用法，然后在以后提供有关缓解浏览器支持问题的建议（请参阅 [处理浏览器支持](#处理浏览器支持)).
+我们将研究 `<input type="week">` 的基本和更复杂的用法，然后在以后提供有关缓解浏览器支持问题的建议（请参阅 [处理浏览器支持](#处理浏览器支持)).
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/week/index.md:110]:
```suggestion
我们将研究 `<input type="week">` 的基本和更复杂的用法，然后在以后提供有关缓解浏览器支持问题的建议（请参阅[处理浏览器支持](#处理浏览器支持)）。
```

File: files/zh-cn/web/html/element/input/week/index.md [hunk 3]
```diff
@@ -160,91 +164,102 @@ input + span {
...
+  content: "✓";
   padding-left: 5px;
 }
 ```
 
-结果是，在 2017 年 W01 到 W52 之间只有几周才被视为有效，并且可以在支持的浏览器中选择。
+结果是，在支持的浏览器中，2017 年 W01 到 W52 之间只有几周才被视为有效，并且可以选择。
 
-### 使周值成为必需
+### 使星期值成为必需
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/week/index.md:182]: 这样是不是更好些：

```suggestion
### 使星期值成为必需值
```

File: files/zh-cn/web/html/element/input/week/index.md [hunk 4]
```diff
@@ -160,91 +164,102 @@ input + span {
...
-      <input type="submit" value="Submit form">
+    <input type="submit" value="Submit form" />
   </div>
 </form>
 ```
 
 如果您尝试提交不带任何值的表单，浏览器将显示错误。 现在尝试使用示例：
 
-{{EmbedLiveSample('Making_week_values_required', 600, 120)}}
+{{EmbedLiveSample('使星期值成为必需', 600, 120)}}
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/week/index.md:209]:
```suggestion
{{EmbedLiveSample('使星期值成为必需值', 600, 120)}}
```

**Review by yin1999 — APPROVED** [review]

File: files/zh-cn/web/html/element/input/week/index.md [hunk 5]
```diff
@@ -274,65 +289,65 @@ input + span {
...
+  for (let i = 1; i <= 52; i++) {
+    const option = document.createElement('option');
+    option.textContent = (i < 10) ? `0${i}` : i;
     weekSelect.appendChild(option);
   }
 }
 ```
 
-> **备注：** 请记住，有些年份有 53 周（请参阅 [每年的周数](https://en.wikipedia.org/wiki/ISO_week_date#Weeks_per_year))！ 在开发生产应用程序时，您需要考虑到这一点。
+> **备注：** 请记住，有些年份有 53 周（请参阅[每年的周数](https://en.wikipedia.org/wiki/ISO_week_date#Weeks_per_year))！ 在开发生产应用程序时，需要考虑到这一点。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/input/week/index.md:350]:
```suggestion
> **备注：** 请记住，有些年份有 53 周（请参阅[每年的周数](https://en.wikipedia.org/wiki/ISO_week_date#Weeks_per_year))！在开发生产应用程序时，需要考虑到这一点。
```

File: files/zh-cn/web/html/element/input/week/index.md [hunk 6]
```diff
@@ -274,65 +289,65 @@ input + span {
...
+  for (let i = 1; i <= 52; i++) {
+    const option = document.createElement('option');
+    option.textContent = (i < 10) ? `0${i}` : i;
     weekSelect.appendChild(option);
   }
 }
 ```
 
-> **备注：** 请记住，有些年份有 53 周（请参阅 [每年的周数](https://en.wikipedia.org/wiki/ISO_week_date#Weeks_per_year))！ 在开发生产应用程序时，您需要考虑到这一点。
+> **备注：** 请记住，有些年份有 53 周（请参阅[每年的周数](https://en.wikipedia.org/wiki/ISO_week_date#Weeks_per_year))！在开发生产应用程序时，需要考虑到这一点。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/input/week/index.md:350]:
```suggestion
> **备注：** 请记住，有些年份有 53 周（请参阅[每年的周数](https://zh.wikipedia.org/wiki/ISO週日曆#每年的星期))！在开发生产应用程序时，需要考虑到这一点。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**